From 4525ad346d226ebeb2f4fe645bb44fb95691fa0d Mon Sep 17 00:00:00 2001
From: GB_2 <bfeber@protonmail.com>
Date: Thu, 12 Sep 2019 16:01:46 +0000
Subject: [PATCH 084/152] windowMenu: Animate menu

Most menus have an animation, so add one to the window menu for
consistency and better looks (like !712).

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/717
---
 js/ui/windowMenu.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/windowMenu.js b/js/ui/windowMenu.js
index 58ef841d4..f2fb750bc 100644
--- a/js/ui/windowMenu.js
+++ b/js/ui/windowMenu.js
@@ -224,7 +224,7 @@ var WindowMenuManager = class {
         this._sourceActor.set_position(rect.x, rect.y);
         this._sourceActor.show();
 
-        menu.open(BoxPointer.PopupAnimation.NONE);
+        menu.open(BoxPointer.PopupAnimation.FADE);
         menu.actor.navigate_focus(null, St.DirectionType.TAB_FORWARD, false);
         menu.connect('open-state-changed', (menu_, isOpen) => {
             if (isOpen)
-- 
2.23.0

